core/src/main/java/com/hcl/ecomm/core/servlets/GetWishListItemsServlet.java

```diff
@@ -0,0 +1,67 @@
+package com.hcl.ecomm.core.servlets;
+
+import com.hcl.ecomm.core.services.WishListService;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Component(service = Servlet.class, property = { "sling.servlet.paths=/bin/hclecomm/getWishList",
+        "sling.servlet.method=" + HttpConstants.METHOD_GET, "sling.servlet.extensions=json" })
+public class GetWishListItemsServlet extends SlingSafeMethodsServlet {
+
+    private static final long serialVersionUID = 4016057296495129474L;
+    private static final Logger LOG = LoggerFactory.getLogger(GetWishListItemsServlet.class);
+
+    @Reference
+    private WishListService wishListService;
+
+    protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
+            throws ServletException, IOException {
+        response.setContentType("application/json");
+        response.setCharacterEncoding("UTF-8");
+
+         try {
+            String customerToken = request.getHeader("CustomerToken");
+            JsonObject responseStream = wishListService.getWishListItems(customerToken);
+            LOG.info("responseStream is {}", responseStream.toString());
```

Comment:
Remove Log.info or change it to Log.debug

Reply:
updated to Log.debug